store/search/src/main/java/org/apache/carbondata/store/worker/SearchRequestHandler.java

```diff
@@ -105,8 +105,6 @@ public ShutdownResponse handleShutdown(ShutdownRequest request) {
       }
     } catch (InterruptedException e) {
       throw new IOException(e);
-    } finally {
-      reader.close();
```

Comment:
@kumarvishal09 @ravipesala I think this close operation is needed. Can you check whether this is ok?

Reply:
I think it is needed, otherwise memory and resources won't be cleaned

Reply:
Ok, keep it.